code/game/objects/items.dm

```diff
@@ -563,7 +563,7 @@ GLOBAL_VAR_INIT(embedpocalypse, FALSE) // if true, all items will be able to emb
 
 	if(M.is_eyes_covered())
 		// you can't stab someone in the eyes wearing a mask!
-		to_chat(user, "<span class='warning'>You're going to need to remove [M.p_their()] eye protection first!</span>")
+		to_chat(user, "<span class='warning'>You failed to eye stab, you need to remove [M.p_their()] eye protection first!</span>")

```

Comment:
awkward to say ``eye stab``, maybe consider ``failed to stab at their eyes`` or something along those lines